.storybook/custom/custom.scss

```diff
@@ -308,3 +308,28 @@ body.sb-show-main {
   color: #32363a;
   color: var(--sapTextColor, #32363a);
```

Comment:
Why aren't we using the theme for some of this: https://github.com/SAP/fundamental-styles/blob/master/.storybook/custom/fundamentals.js#L4